source/MaterialXGenGlsl/Nodes/TangentNodeGlsl.cpp

```diff
@@ -25,7 +25,7 @@ void TangentNodeGlsl::createVariables(const ShaderNode& node, GenContext&, Shade
     const int space = spaceInput ? spaceInput->getValue()->asA<int>() : OBJECT_SPACE;
     if (space == WORLD_SPACE)
     {
-        addStageUniform(HW::PRIVATE_UNIFORMS, Type::MATRIX44, HW::T_WORLD_INVERSE_TRANSPOSE_MATRIX, vs);
+        addStageUniform(HW::PRIVATE_UNIFORMS, Type::MATRIX44, HW::T_WORLD_MATRIX, vs);
```

Comment:
Why is this now using the world matrix ?

Reply:
Because, as the commit message explains, tangents are not transformed using the normal matrix:
https://pbr-book.org/3ed-2018/Geometry_and_Transformations/Applying_Transformations#Normals

Reply:
That's a great catch, @pablode, and we'd overlooked that detail in earlier implementations/review of the tangent node.